app/javascript/styles/_uploadWizard.scss

```
@@ -147,3 +146,7 @@
 .prev-next-buttons {
   margin-left: auto;
 }
+
+.margin-left {
```

For consistency with e.g.  `top-margin`.

```suggestion
.left-margin {
```